pandas/tests/io/excel/test_writers.py

```diff
@@ -1162,6 +1162,19 @@ def test_path_local_path(self, engine, ext):
                                        path="foo.{ext}".format(ext=ext))
         tm.assert_frame_equal(result, df)
 
+    def test_merged_cell_custom_objects(self, engine, ext):
+        # see GH-27006
+        mi = MultiIndex.from_tuples([(pd.Period('2018'), pd.Period('2018Q1')),
+                                     (pd.Period('2018'), pd.Period('2018Q2'))])
+        expected = DataFrame(np.random.rand(2, len(mi)), columns=mi)
+        expected.to_excel(self.path)
```

Comment:
Does this fail on master? Tried locally and ran so want to make sure we are testing the right thing

Reply:
I think maybe I am misunderstanding how I am supposed to be testing? If I take the code above from lines 1167-1170 and drop it in a jupyter notebook running in a conda env with pandas 0.24.2 installed, I get the expected `TypeError`.

Do I also need to put that test code into the master branch on my fork and have it fail there? I only put it on the 'period-excel-fix' branch of my fork where I fixed the bug, which passes pytest when I run.

Reply:
Sorry this does fail on master so all good. But yea typically want to run against master

Reply:
ok great